uacp/conn.go

```diff
@@ -26,6 +26,7 @@ const (
 	DefaultSendBufSize    = 0xffff
 	DefaultMaxChunkCount  = 512
 	DefaultMaxMessageSize = 2 * MB
+	DefaultDialTimeout    = time.Second * 10
```

Comment:
Could you change this to `10 * time.Second` so that it's similar to the line above?

Reply:
was going to fix this and noticed we don't use `DefaultDialTimeout` anywhere...
The `Config` + `Dialer` is a bit confusing/inconsistent. Will patch that up as well.

Reply:
Out of curiosity: wouldn't you just need to mock a `net.Conn` ?

Reply:
can you mock a "syn->ack" where the ack never comes back? i guess I'm not clear exactly what the tcp dial timeout covers. i generally think of the `net.Conn` as a "high" level interface.

Reply:
Right. Well, that would be a fun exercise for a different time.